src/components/modes/teacher/ImageUpload.tsx

```diff
@@ -32,6 +32,8 @@ const ImageUploadContainer = styled('div')(() => ({
   marginBottom: 1,
 }));
 
+const SubTypography = styled(Typography)(() => ({ color: '#383838' }));
```

Comment:
Even this should work, but no worries.
```suggestion
const SubTypography = styled(Typography)({ color: '#383838' });
```